src/main/java/io/getunleash/metric/UnleashMetricServiceImpl.java

```diff
@@ -28,6 +37,14 @@ public UnleashMetricServiceImpl(
         this.started = LocalDateTime.now(ZoneId.of("UTC"));
         this.unleashConfig = unleashConfig;
         this.metricSender = metricSender;
+        this.maxInterval =
+                Integer.max(
+                        20,
+                        300
+                                / Integer.max(
+                                        Long.valueOf(unleashConfig.getSendMetricsInterval())
+                                                .intValue(),
```

Comment:
According to Long#intValue method:
```java
    /**
     * Returns the value of this {@code Long} as an {@code int} after
     * a narrowing primitive conversion.
     * @jls 5.1.3 Narrowing Primitive Conversion
     */
    public int intValue() {
        return (int)value;
    }
```

So this should be safe:
```suggestion
                                / Integer.max(
                                        (int) unleashConfig.getSendMetricsInterval(),
```